### PR TITLE
Portfolio: Desktop Version

### PR DESCRIPTION
**Pull request by Besufekad-HAZ**
- Added a responsive desktop version
- Used media queries to make the page responsive on big screens

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -3,65 +3,59 @@
...
-  align-items: center;
-  background-image: url(img/image_geometry_1.png);
-  display: flex;
-  justify-content: space-between;
-  background-repeat: no-repeat;
-  background-size: contain;
-  height: 70px;
+body {
+  background-color: #f9f9fa;
 }
```
Comment (OyePriscilla) [inline styles.css]: - [ ] Kindly remove the horizontal overflow in order to enhance the user experience. 😃 

## Your Design

b11

File: styles.css [hunk 2]
```diff
@@ -162,293 +132,559 @@ a:active {
...
+  }
+
+  #about-section {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    margin-right: 100px;
+    margin-top: 210px;
+    margin-bottom: 150px;
+  }
```
Comment (OyePriscilla) [inline styles.css]: - [ ] Please kindly apply margin to the` about me section` so as to resemble the original template. 👍 

## Your Design

b10

## Figma Design

b8

File: styles.css [hunk 3]
```diff
@@ -162,293 +132,559 @@ a:active {
...
+  .forbackground .formbody {
+    margin: auto;
+    text-align: justify;
+    width: 35vw;
+  }
+
+  .name_and_email {
+    width: 35vw;
+    display: flex;
+  }
```
Comment (OyePriscilla) [inline styles.css]: - [ ] Please take note that you need to add margin in-between the input boxes according to the design template. As developers, when we pay attention to details; it shows that we can deliver as expected.😃 

## Your Design

b9

## Figma Design

b7

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Besufekad-HAZ   😃 , 

Good job so far!
There are little issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
LInters checks passed ✔️ 
GitHub flow observed ✔️ 

## Required Changes ♻️
Kindly check the inline comments below for the required changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @Besufekad-HAZ, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**